.eslintignore

```diff
@@ -1,5 +1,5 @@
 # Output directories
-node_modules
+node_modules/**
```

Comment:
What is the difference here? I was under the impression a bare-name line in gitignore syntax ignored files/folders with that name (which implicitly includes all contained files and subfolders), and so the trailing `/**` here and elsewhere in this file shouldn't be necessary. Am I missing something?